index.html

```diff
@@ -166,8 +166,23 @@
                 //Not Working ATM
                 combat: function() {
                     var enemy = this
-                    setInterval (function() {
-                       if (enemy.x > Crafty('PlayerCharacter').x) {
+
+                    enemy.bind('Moved', function(movement) {
```

Comment:
binding this move event did not make sense inside the setInterval -- you don't want to bind events in a loop like that (or if your interval runs 100 times, the "Moved" callback will get called 100 times per movement)